requirements.txt

```diff
@@ -1,2 +1,4 @@
-us
-requests
\ No newline at end of file
+us~=2.0.2
+requests
+pandas~=1.2.4
+mysql-connector-python~=8.0.25
```

Comment:
Looks like you learned to control the version also on these. Very important, as new versions of Pip modules can break application functionality. To change versions on Pip modules after the application is in use, you would want to upgrade the module in a testing environment, confirm good operation, then merge the `requirements.txt` changes with new versions into the `master` / `production` branch.